src/webgl/material.js

```diff
@@ -127,11 +130,25 @@ p5.prototype.loadShader = function(
  * // the 'varying's are shared between both vertex & fragment shaders
  * let varying = 'precision highp float; varying vec2 vPos;';
  *
+ * Shaders can alter the positioning of shapes drawn with them.
+ * To ensure consistency in rendering, it's recommended to use the vertex shader, as in the example below.
```

Comment:
Also, super minor wording: it might flow a bit better if you say "the vertex shader shown in the example below."